azure-pipelines.yml

```diff
@@ -423,7 +423,7 @@ jobs:
 
   - script: |
       cd examples
-      pip install openml pandas
+      pip install -e ".[examples]"
```

Comment:
Dang, this fails because you are in the examples directory...

Reply:
Thanks for the catch!